include/orng.hpp

```diff
@@ -94,6 +107,16 @@ CONTRACT orng: public eosio::contract {
     };
     using config_table_type = eosio::multi_index<"config.a"_n, config_a>;
 
+    // Config table
+    TABLE sigpubkey_config
+    {
+        uint64_t chance_to_switch;
+        uint64_t active_key_index;
+        uint64_t available_key_counter;
+    };
+    using pubconfig_table_type = eosio::singleton<"pubconfig.a"_n, sigpubkey_config>;
```

Comment:
can we rename `pubconfig_table_type` to `sigpubconfig_table_type`, and `pubconfig.a` to `sigpubconf.a` please. Having the sig in front keeps my mind on topic a little easier